src/skia/Font.cpp

```diff
@@ -352,6 +413,19 @@ fontarguments
         actually be indexed collections of fonts.
         )docstring",
         py::arg("collectionIndex"))
+    .def("setPalette", &SkFontArguments::setPalette,
+        R"docstring(
+        Specify a palette to use and overrides for palette entries.
+        )docstring",
+        py::arg("palette"))
+    .def("setPalette",
```

Comment:
There are some `setPaletteIndex/getPaletteIndex` methods in other classes. Perhaps do both?

Reply:
I also wonder whether this should be a read/write property instead.